composer.json

```diff
@@ -11,12 +11,13 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.2 || ^8",
```

Comment:
This is bad idea, PHP 8 is not even close to getting an alpha release and more BC breaks are coming.

Reply:
I was on the fence myself. but this matches the requirements of ramsey/uuid which this package is based on, so it should be safe.

ref: https://github.com/ramsey/uuid/blob/b3c39208757f739e3e53c391e21dc7fc6afa1cf5/composer.json#L13

Reply:
The builds consistently fail for nightlies on Travis CI, but when I completely ignore platform requirements locally and tweak a few things (e.g., Reflection APIs), then everything runs smoothly on 8.0.0-dev. My hope is that, over time, more projects will start using `|| ^8` and addressing the issues in their libraries that break on PHP 8 (and are known to be stable right now, such as Reflection APIs). One day, it will be nice to see the nightlies magically start passing. 😉

Reply:
If anyone wants to juggle with PHP 8 before at least first beta is out they can use `--ignore-platform-reqs` made for that purpose. But allowing PHP 8 in stable packages at this point is really bad idea, you're claiming to be compatible with something that doesn't even exist yet (remember PHP 6 book? 😃).

Reply:
I don't understand why it's a problem in stable packages. No one should be running PHP 8 in production, so having `|| ^8` in `composer.json` shouldn't cause any problems.

PHP 6 was an entirely different situation.